### PR TITLE
feat: update docs to outline install process

### LINKED ISSUE #134: [Docs] Update hamlet-cli README to reflect registration on pypi
## Documentation Topic
hamlet-cli repo README

## Description
Current documentation suggests installing via cloning of github repo. However, with the integration of the hamlet-cli with pypi, a simpler install experience would be to simply use `pip install hamlet-cli`

## Audience
Users of hamlet deploy looking to integrate it into their own pipelines.

### PR DESCRIPTION
**Pull request by roleyfoley**
## Intent of Change

- Documentation (new or update to existing)

## Description

Updates the repo and package README files to provide more information on the install process and to redirect audiences between the different pages

## Motivation and Context

Make it easier to find out what hamlet does on pypi and how to get started with 
Move users to the user docs instead of using the development install process 

closes #134 

## How Has This Been Tested?

Tested locally

### Prerequisite PRs:

- None

### Dependent PRs:

- None

### Consumer Actions:

- None

### CODE REVIEW

File: hamlet-cli/README.md [hunk 1]
```diff
@@ -1,5 +1,84 @@
...
+## Docs
+
+To read more about hamlet and what it can do head to our docs site https://docs.hamlet.io/
+
+## Installation
+
+Hamlet Deploy is made up of a couple of parts:
+
+- engine - The engine is the core of hamlet and defines how infrastructure is deployed along with understanding the context of your application defined in a CMDB. The engine creates contracts and supporting documents which describes a task to complete. The engine is a java based app using the freemarker template engine.
+- executor - The executor providers the user interface to hamlet along with the executing what the contracts provided by the engine. This is a mix of bash scripts and python code ( this cli )
```
Comment (ml019) [inline hamlet-cli/README.md:25]:
```suggestion
- executor - The executor providers the user interface to hamlet along with executing the contracts provided by the engine. This is a mix of bash scripts and python code ( this cli )
```

File: hamlet-cli/README.md [hunk 2]
```diff
@@ -1,5 +1,84 @@
...
+
+Before using the cli a few OS level requirements need to be installed. This process differs from OS to OS so we are just including the packages and the websites if they are available. The packages need to be available on the PATH of your cli
+
+#### Minimum
+
+These packages will give you base access to start using hamlet
+
+| Name   | Install Link                                 | Version           |
+|--------|----------------------------------------------|-------------------|
+| Java   | https://openjdk.java.net/install/            | 8/1.8 Required    |
```
Comment (ml019) [inline hamlet-cli/README.md:43]: I think this now needs to be 11? Need to confirm with Kseniya if new wrapper still runs on 8

**Review by ml019 — CHANGES_REQUESTED** [review]
It is getting simpler!

A couple of small changes.

File: hamlet-cli/README.md (hunk 2 above)
Comment (roleyfoley) [reply hamlet-cli/README.md:43]: Not just yet. The PR hasn't been tested and @kshychko has put it back to a draft this morning.

**Comment by roleyfoley** [comment]
Closes #134

**Review by ml019 — APPROVED** [review]

**Pull request merged**